browser/ui/webui/brave_wallet/wallet_common_ui.cc

```diff
@@ -1,7 +1,7 @@
-/* Copyright (c) 2021 The Brave Authors. All rights reserved.
- * This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this file,
- * You can obtain one at http://mozilla.org/MPL/2.0/. */
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
```

Comment:
please keep 2021 as that is a year when the file was created

Reply:
fixed